src/lib/utilities/decode-payload-lazy.ts

```diff
@@ -0,0 +1,23 @@
+import type { Payload } from '$types';
```

Comment:
This was named lazy because I was being lazy fighting with jest about imports not working correctly with the other parts of the decode-payload file.  Definitely needs to be remediated differently.